assets/performanceprofile/tuned/openshift-node-performance

```diff
@@ -109,9 +109,10 @@ vm.dirty_background_ratio=3
 vm.swappiness=10
 
 # also configured via a sysctl.d file
-# placed here for better reflection of the change
+# placed here for documentation purposes and commented out due
+# to a tuned logging bug complaining about duplicate sysctl
```

Comment:
Please reference the exact TuneD bug here.  Otherwise, LGTM.

Reply:
Done